azdo-task/README.md

```diff
@@ -73,7 +73,7 @@ In the example above, the devcontainer-build-run will perform the following step
 | subFolder                 | false    | Use this to specify the repo-relative path to the folder containing the dev container (i.e. the folder that contains the `.devcontainer` folder). Defaults to repo root                                                                                          |
 | runCmd                    | true     | The command to run after building the dev container image                                                                                                                                                                                                        |
 | env                       | false    | Specify environment variables to pass to the dev container when run                                                                                                                                                                                              |
-| push                      | false    | One of: `never`, `filter`, `always`. When set to `filter`, the image is pushed if the `sourceBranchFilterForPush`, `buildReasonsForPush`, and `pushOnFailedBuild` conditions are met. Defaults to `filter` if `imageName` is set, `never` otherwise.             |
+| push                      | false    | One of: `never`, `filter`, `always`. When set to `filter`, the image if pushed if the `sourceBranchFilterForPush`, `buildReasonsForPush`, and `pushOnFailedBuild` conditions are met. Defaults to `filter` if `imageName` is set, `never` otherwise.             |
```

Comment:
Incorrect change. The `if` doesn't make sense here.

Reply:
Thank you, should be resolved now